sympy/utilities/tests/test_iterables.py

```diff
@@ -732,3 +732,14 @@ def test_ordered_partitions():
                 sum(1 for p in f(i, j, 1)) ==
                 sum(1 for p in f(i, j, 0)) ==
                 nT(i, j))
+
+
+def test_rotations():
+    assert list(rotations([10,-20,20])) == [[10, -20, 20], [-20, 20, 10], [20, 10, -20]]
```

Comment:
You can thin this out a bit: perhaps test `'ab'`, `range(3)`, and one of those with `dir=-1`.